docs/source/_json/breadcrumbs.json

```diff
@@ -0,0 +1,19 @@
+GET /plone/front-page/@components/breadcrumbs
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+[
+  {
+    "data": {
+      "items": [
+        {
+          "title": "Welcome to Plone", 
+          "url": "http://localhost:55001/plone/front-page"
+        }
+      ]
+    }, 
+    "id": "breadcrumbs"
```

Comment:
@ebrehault as @lukasgraf mentioned I would recommend to use the JSON-LD "@id" notation with the URL as value here. Do we use "id" for anything?